src/renderer/reducers/settings.js

```diff
@@ -338,6 +358,8 @@ export const dismissedBannerSelectorLoaded = (bannerKey: string) => (state: Stat
 export const hideEmptyTokenAccountsSelector = (state: State) =>
   state.settings.hideEmptyTokenAccounts;
 
+export const lastSeenDeviceSelector = (state: State) => state.settings.lastSeenDevice;
```

Comment:
to me there is no reason we need this "raw" selector but i'm not seeing the selector that tells if blue is on or off 🤔

Reply:
ahah still a wip